### PR TITLE
iOS-friendly audio and animation tweaks, updated favicon

### LINKED ISSUE #33: iOS/Safari: SVG notes do not animate
There is no animation on SVG notes in Safari and iOS. Assigning to @franknoirot after I struck out debugging.

image

### LINKED ISSUE #34: iOS no audio playback when phonogram activated
Getting mixed results in Safari, but no audio is playing back in iOS period. :(

### LINKED ISSUE #35: New favicon option(s)
I'll close this is EF/AF/CL come up with something before Tuesday, but LS would like other favicon options. I do not blame her 😁

### LINKED ISSUE #36: Autoplay audio support?
Let's TB on audio autoplay -- LS would like this but I think it's a security no-no (at least in Chrome). I can update state/toggle for this (and added autoplay attribute onto the tag), but no actual audio plays.

### PR DESCRIPTION
**Pull request by franknoirot**
Resolves the following issues:
- Fixes #33: the `fill` property cannot be animated for `<path>` elements on Safari, but the `opacity` property can, so I used that instead.
- Fixes #34: this was a permissions issue, needed to handle the Promise returned from the `play()` function call within a try-catch statement, used [MDN example code](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/play#example) and it worked fine.
- Fixes #35 pending approval: I made a version of the [Favicon in Figma](https://www.figma.com/file/lQ7CjNpS2EoFp27o3xR2WI/Concepts-and-Wireframes?node-id=2%3A162) which filled in the L and G to give them more weight on that small scale, then set them on a white background for higher contrast. Used favicon-generator.org to create the various sizes and theme info, copy-pasted the code into the `Layout` component.
- Resolves #36 (Unable to fix): The use case of autoplay on site load is no longer supported by any browser. Some interaction is a required to begin audio playback, so the play button is likely the best solution. Will comment into the issue with citations before closing.

### CODE REVIEW

**Review by robpetrin — COMMENTED** [review]
Hi @franknoirot -- changes all look good, though I'd be curious how you came to your conclusions on the note animation and iOS playback fixes. Your solutions make sense, and even though I was working on a tight deadline, I was stonewalled on determining those fixes.

Commenting approval but not closing until you get LS approval on favicon asset.

File: components/Layout/Layout.js [hunk 1]
```diff
@@ -6,7 +6,23 @@ export default function Layout(props) {
...
+            <link rel="apple-touch-icon" sizes="152x152" href="favicon/apple-icon-152x152.png"/>
+            <link rel="apple-touch-icon" sizes="180x180" href="favicon/apple-icon-180x180.png"/>
+            <link rel="icon" type="image/png" sizes="192x192"  href="favicon/android-icon-192x192.png"/>
+            <link rel="icon" type="image/png" sizes="32x32" href="favicon/favicon-32x32.png"/>
+            <link rel="icon" type="image/png" sizes="96x96" href="favicon/favicon-96x96.png"/>
+            <link rel="icon" type="image/png" sizes="16x16" href="favicon/favicon-16x16.png"/>
+            <link rel="manifest" href="favicon/manifest.json"/>
+            <meta name="msapplication-TileColor" content="#ffffff"/>
+            <meta name="msapplication-TileImage" content="favicon/ms-icon-144x144.png"/>
+            <meta name="theme-color" content="#ffffff"/>
```
Comment (franknoirot) [inline components/Layout/Layout.js:25]: All this is essentially boilerplate from favicon-generator.org, only made them JSX-friendly closed tags and put the icons into a `favicon/` folder.

File: lib/AppContext.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// This app context is no longer used, but remains here as an example of how to create a context wrapper and reducer.
```
Comment (franknoirot) [inline lib/AppContext.js:1]: I didn't think it was worth it to use the context anymore, but I'm definitely gonna forget best practices when I need to create one in the future.

File: public/favicon/manifest.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+ "name": "App",
```
Comment (franknoirot) [inline public/favicon/manifest.json:2]: This is still boilerplate, could be given a more specific name if we'd like.

**Review by franknoirot — COMMENTED** [review]
Just adding in some context notes.

**Comment by franknoirot** [comment]
@robpetrin here are some of my thoughts for solving these issues:

## Note animation 

This would have been really tricky for you to know. I've done a lot of attempts to animate SVGs, especially for stuff like my [map animator tool](https://franknoirot.co/work/map-animator). I made this [little toggle animation loop](https://codepen.io/franknoirot/pen/abzymrq) I was really proud of back in the day and was disappointed by how broken it is on Safari and Firefox, so I investigated what other properties of SVGs don't work in transitions/animations. Chrome supports most, most other browsers support very little.

## Playback issues on iOS
This one you probably could've hacked your way to like I did, but it required comfort with the component to really rip out some logic to figure out where the break was happening. With my way overengineered `AppContext` and `AppReducer` in place I bet it made investigating a lot harder for you.

First thing I did was add the `controls` property to the audio element to make the player visible, then tested pressing play directly on there in Safari. Since that worked I knew it was indeed a permissions thing with Safari, so I googled these three phrases: "safari get user audio permission", "safari request user audio permission", and "safari custom play button not working" when those first two led me to examples of getting access to a user's microphone.

I was led back to the MDN example page for `HTMLMediaElement.play()` and I was confused why my code wouldn't work the same way, so I just tried adapting their example as directly as possible and it worked. What I think my code was doing wrong is it wasn't handling a case that must occur initially after play button press where the Promise returned from the `play()` method isn't valid. I believe the code to call that method was occurring _before_ the event had truly fired on Safari, so the user interaction didn't "count". By wrapping that attempt to play in a try-catch block, it allowed the method to try firing again at the end of the click.

**Comment by franknoirot** [comment]
Just received LS approval on the favicon update @robpetrin, merging.

**Pull request merged**